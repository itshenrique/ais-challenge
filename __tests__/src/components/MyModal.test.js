/**
 * @format
 */
import 'react-native';
import React from 'react';
import {Text} from 'react-native';
import Modal from '../../../src/components/MyModal';
import {shallow} from 'enzyme';

describe('Modal Tests', () => {
  const modalVisible = true;
  const closeModal = jest.fn();
  const component = shallow(<Modal />);

  component.setProps({
    modalVisible,
    closeModal,
  });

  it('render modal', async () => {
    expect(component).toMatchSnapshot();
  });
});
