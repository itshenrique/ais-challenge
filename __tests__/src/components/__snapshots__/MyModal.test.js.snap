// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Tests render modal 1`] = `ShallowWrapper {}`;
